src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2870,4 +2870,10 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="TextExtractor_Languages.Header" xml:space="preserve">
     <value>Preferred language</value>
   </data>
+  <data name="GeneralPage_HideSystemTrayIcon.Description" xml:space="preserve">
+    <value>The settings menu is accessible by running the PowerToys executable again</value>
```

Comment:
Does this message sound better and give the user enough information? Would there be a preferable way to communicate this?